### PR TITLE
damage_mult applies to sunder and penetration

### PR DESCRIPTION
**Pull request by Waselon**
## About The Pull Request

title

## Why It's Good For The Game

both values are quite important, this is mainly to fix quick-fire shotguns applying sunder too fast.

## Changelog
:cl:
fix: damage_mult applies to sunder and penetration
/:cl:

### CODE REVIEW

**Comment by QualityVan** [comment]
Sundering I can understand, but penetration seems unnecessary.

**Comment by FIXTHERADBOXPLEASE** [comment]
Keep in mind penetration already uses low numbers, so the multiplier won't affect it much.

**Comment by Wayland-Smithy** [comment]
> penetration already uses low numbers

Except where it doesn't. Either way this will be a far reaching buff to who knows how many weapons/ammo.

**Comment by Waselon** [comment]
> > penetration already uses low numbers
> 
> Except where it doesn't. Either way this will be a far reaching buff to who knows how many weapons/ammo.

mostly for seasonal weapons but who cares about seasonals, right?

**Comment by FIXTHERADBOXPLEASE** [comment]
> Except where it doesn't. Either way this will be a far reaching buff to who knows how many weapons/ammo.

None of the high-penetration guns have a negative damage multiplier IIRC.

**Comment by Losenis** [comment]
T39 confirmed obsolete

**Comment by KerbalCitizen** [comment]
One day, it will be insanity to run anything but slugs in T39. The slugs hitting about as hard as a boot knife and only being used for the staggerstun. I use T39 with slugs a lot, the damage is already pretty meh. I am in coping mode. Please send help.

**Comment by Lumipharon** [comment]
Applying to pen means the damage mult is effectively being applied twice.
Sunder seems fine though.

**Comment by Pariah919** [comment]
why not make it a seperate stat

File: code/modules/projectiles/gun_system.dm [hunk 1]
```diff
@@ -1591,6 +1591,8 @@
 /obj/item/weapon/gun/proc/apply_gun_modifiers(obj/projectile/projectile_to_fire, atom/target, firer)
 	projectile_to_fire.shot_from = src
 	projectile_to_fire.damage *= damage_mult
+	projectile_to_fire.penetration *= damage_mult
```
Comment (Waselon) [inline code/modules/projectiles/gun_system.dm:1594]:
```suggestion
```

Comment (Lumipharon) [inline code/modules/projectiles/gun_system.dm:1594]:
```suggestion
```

**Pull request closed without merging**